library/src/lib/calendar/calendar-views/calendar-day-view/calendar-day-view.component.ts

```diff
@@ -0,0 +1,521 @@
+import {
+    AfterViewChecked,
+    Component,
+    ElementRef,
+    EventEmitter,
+    HostBinding,
+    Input, OnChanges,
+    OnInit,
+    Output, SimpleChanges,
+    ViewEncapsulation
+} from '@angular/core';
+import { CalendarI18n } from '../../i18n/calendar-i18n';
+import { FdDate } from '../../models/fd-date';
+import { CalendarCurrent } from '../../models/calendar-current';
+import { CalendarType, DaysOfWeek } from '../../calendar.component';
+import { CalendarDay } from '../../models/calendar-day';
+import { CalendarService } from '../../calendar.service';
+import { FdRangeDate } from '../../models/fd-range-date';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+/** Component representing the day view of the calendar. */
+@Component({
+    selector: 'fd-calendar-day-view',
+    templateUrl: './calendar-day-view.component.html',
+    styleUrls: ['./calendar-day-view.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class CalendarDayViewComponent implements OnInit, AfterViewChecked, OnChanges {
+
```

Comment:
Organize the inputs with inputs, outputs with outputs... etc

Reply:
It can't pass through pre commit lint cause inputs declare functions.

Reply:
For the functions, that's fine. But for the rest should be fine